mlflow/utils/time_utils.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+from pytz import reference
+
+def conv_longdate_to_str(longdate, local_tz=True):
```

Comment:
Mind adding a unit test for this method to confirm its behavior? Could be a simple one like
https://github.com/mlflow/mlflow/blob/5adc84ce5e4270a08846975b4e3952bc99a3715f/tests/utils/test_string_utils.py#L12